### PR TITLE
fixing broken doc links

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Patch fix for broken doc references

### CODE REVIEW

File: website/docs/d/floating_ip.html.markdown [hunk 1]
```diff
@@ -71,4 +71,6 @@ The `vpc_reference`, `vm_nic_reference`, `external_subnet_reference` attributes
 
 * `kind`: - The kind name (Default value: project).
 * `name`: - the name.
-* `uud`: - the UUID.
\ No newline at end of file
+* `uud`: - the UUID.
```
Comment (siddharth-kulshrestha) [inline website/docs/d/floating_ip.html.markdown]:
```suggestion
* `uuid`: - the UUID.
```

**Review by siddharth-kulshrestha — APPROVED** [review]

File: website/docs/d/floating_ip.html.markdown (hunk 1 above)
Comment (Abhishekism9450) [reply website/docs/d/floating_ip.html.markdown]: Fixed in other places too.

Comment (bhati-pradeep) [inline website/docs/d/floating_ip.html.markdown]: uuid

Comment (bhati-pradeep) [inline website/docs/d/floating_ip.html.markdown]: uuid

**Review by bhati-pradeep — CHANGES_REQUESTED** [review]
LGTM. Minor changes requested

**Review by bhati-pradeep — APPROVED** [review]

**Review by abhimutant — APPROVED** [review]

**Pull request merged**